de.dlr.sc.virsat.server.test/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,10 @@ Bundle-Version: 4.11.0.qualifier
 Fragment-Host: de.dlr.sc.virsat.server;bundle-version="0.0.1"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: DLR (German Aerospace Center)
-Require-Bundle: org.junit
+Require-Bundle: org.junit,
+ org.eclipse.egit.core,
+ org.hamcrest.library,
+ org.eclipse.team.svn.core
 Export-Package: de.dlr.sc.virsat.server.test
 Automatic-Module-Name: de.dlr.sc.virsat.server.test
+Import-Package: de.dlr.sc.virsat.project.test

```

Comment:
This is because Tycho maven resolves fragments differently then an eclipse application. If you access a class from one fragment into another fragment, it needs to be imported implicitly via the IMPORT-PACKAGE. Otherwise the maven/tycho will run into a compilation issue.